sashimi-webapp/src/database/data-modifier/dataAdd.js

```diff
@@ -92,9 +148,16 @@ export default class dataAdd {
         sqlCommands.getMaxFileId()
           .then((maxId) => {
             const newFileId = maxId + 1;
-            return createNewFile(organizationId, filePath, folderId, newFileId)
-            .then(data => resolve(data))
-            .catch(err => reject(err));
+            // search all possible identical fileNames
+            return sqlCommands.exactSearchStartFileNameInFolder(
+              constants.DEFAULT_FILE_NAME_OMIT_FILE_TYPE, filePath)
```

Comment:
I notice that `exactSearchStartFileNameInFolder` is not doing anything with the first parameter.


`searchString` is not being used. 
```javascript
this.exactSearchStartFileNameInFolder = function exactSearchStartFileNameInFolder(searchString, filePath) {
    if (typeof Promise === 'function') {
      return new Promise((resolve, reject) =>
        alasql.promise([stringManipulator.stringConcat('SELECT ', constants.HEADER_FILE_MANAGER_FILE_NAME,
                                                       ' FROM ', constants.ENTITIES_FILE_MANAGER,
                                                       ' WHERE ', constants.HEADER_FILE_MANAGER_PATH,
                                                       ' LIKE "', filePath, '%"',
                                                       ' ORDER BY ', constants.HEADER_FILE_MANAGER_FILE_NAME,
                                                       ' ASC')])
        .then(data => resolve(getArray(data)))
        .catch(sqlError => reject(sqlError))
      );
    } else {
      throw new exceptions.PromiseFunctionNotDefined();
    }
};
```